repository/git.go

```diff
@@ -116,7 +157,10 @@ func (r *Git) writeConfig() (err error) {
 	if err != nil {
 		return
 	}
-	s := "[credential]\n"
+	s := "[user]\n"
+	s += "name = Konveyor Dev\n"
+	s += "email = konveyor-dev@googlegroups.com\n"
```

Comment:
This is the piece we need to chat with @jortel and @rromannissen about how we want to handle. My initial thought is that this user block could be a fallback option in the case the stored identity doesn't have a name + email set.